sympy/core/cache.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import print_function, division
 
 from distutils.version import LooseVersion as V
-
+from sympy.core.compatibility import lru_cache
```

Comment:
This should not be necessary. `lru_cache` is imported on line 43 already.

Reply:
Yes , i will remove it.